### PR TITLE
Add deployment configuration for the Angular example (Firebase)

### PR DESCRIPTION
**Pull request by borja-munoz**
Add the configuration for deploying the Angular example to Firebase

### CODE REVIEW

File: deck.gl/examples/pure-js/angular/firebase.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (VictorVelarde) [inline deck.gl/examples/pure-js/angular/firebase.json:1]: If we're gonna keep the pub. info here, you might want to add a small description of the deployment process to readme, like we have in https://github.com/CartoDB/carto-react-template/blob/master/DEVELOPERS.md#deploying-the-sample-app

**Review by VictorVelarde — APPROVED** [review]
👍🏻

File: deck.gl/examples/pure-js/angular/firebase.json (hunk 1 above)
Comment (borja-munoz) [reply deck.gl/examples/pure-js/angular/firebase.json:1]: Deployment instructions added

**Review by VictorVelarde — APPROVED** [review]
:+1

**Pull request merged**